ir/attrs.h

```diff
@@ -18,8 +22,8 @@ class ParamAttrs final {
 
   ParamAttrs(unsigned bits = None) : bits(bits) {}
 
-  uint64_t derefBytes; // Dereferenceable
-  uint64_t derefOrNullBytes; // DereferenceableOrNull
+  uint64_t derefBytes = 0; // Dereferenceable
+  uint64_t derefOrNullBytes = 0; // DereferenceableOrNull
```

Comment:
dereferenceable and dereferenceable_or_null cannot be zero (https://godbolt.org/z/WWdEs7 , https://godbolt.org/z/f8Kfz9), so it is sound to have zero as a value to indicate that there's no 'dereferenceable' attr.